models/comment.js

```diff
@@ -0,0 +1,22 @@
+module.exports = (sequelize, DataTypes) => {
+  const Comment = sequelize.define('Comment', {
+    comment: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    articleId: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+    },
+  });
+  Comment.associate = (models) => {
+    Comment.belongsTo(models.article, {
```

Comment:
Change the `article` here to the appropriate case we agreed on.

Also, pull from develop, and uncomment the commented section in the `Article` model

Reply:
Feedback implemented